### PR TITLE
zathura:  add page

### PR DESCRIPTION
**Pull request by gsobell**
Not sure which commands  and shortcuts to include/exclude, and style could use some polishing.

List for reference can be found here: https://defkey.com/zathura-shortcuts

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/zathura.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# zathura
+
+> zathura is a vim-like modal document viewer, with an integrated command line
```
Comment (navarroaxel) [inline pages/linux/zathura.md:3]:
```suggestion
> A vim-like modal document viewer, with an integrated command line.
```

File: pages/linux/zathura.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+> zathura is plugin based; make sure a backend is installed (poppler, PostScript or DjVu)
+> More information: <https://pwmt.org/projects/zathura/>.
+
+- Navigate left/up/down/right:
+
+`<H|J|K|L> or arrow keys`
+
+- Rotate:
+
+`<r>`
```
Comment (navarroaxel) [inline pages/linux/zathura.md:13]:
```suggestion
`r`
```

File: pages/linux/zathura.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ 
+- Invert Colors:
+
+`<Ctrl-r>`
```
Comment (navarroaxel) [inline pages/linux/zathura.md:17]:
```suggestion
`Ctrl + r`
```

File: pages/linux/zathura.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- Search for text:
+
+`</>`
```
Comment (navarroaxel) [inline pages/linux/zathura.md:21]:
```suggestion
`/`
```

File: pages/linux/zathura.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+- Create/delete/list bookmarks:
+
+`{{:bmark|:bdelete|:blist}} bookmark-name`
```
Comment (navarroaxel) [inline pages/linux/zathura.md:25]: We use `snake_case` for placeholders.
```suggestion
`:{{bmark|bdelete|blist}} {{bookmark_name}}`
```

**Review by navarroaxel — COMMENTED** [review]
You should add examples about how to star Zathura, you can check `vim` and `nano` pages for reference.

File: pages/linux/zathura.md (hunk 1 above)
Comment (gsobell) [reply pages/linux/zathura.md]: Added

**Comment by gsobell** [comment]
> You should add examples about how to star Zathura, you can check `vim` and `nano` pages for reference.

@navarroaxel  what do you mean by this? checked vim and nano pages

**Comment by gsobell** [comment]
> The [build](https://github.com/tldr-pages/tldr/actions/runs/1183209370) for this PR failed with the following error(s):
> 
> ```
> pages/linux/zathura.md:13: TLDR014 Page should contain no trailing whitespace
> ```
> 
> Please fix the error(s) and push again.

No trailing whitespace on line 13, fails to pass anyway.

**Comment by sbrl** [comment]
It looks like it was the line below :-)

Looks like the line numbers tldr-bot uses count from 0, but most text editors I know count from 1.  Bug, @owenvoke?

File: pages/linux/zathura.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+`Ctrl + R`
+
+- Search for text:
+
+`/`
+
+- Create/delete/list bookmarks:
+
+`:{{bmark|bdelete|blist}} {{bookmark_name}}`
```
Comment (sbrl) [inline pages/linux/zathura.md:25]: You can list a specific bookmark?

Comment (gsobell) [reply pages/linux/zathura.md:25]: `:blist` is used without arguments, my bad. Is there a way yo include it in this entry (I don't think it rates it's own)?

**Comment by navarroaxel** [comment]
> @navarroaxel what do you mean by this? checked vim and nano pages

I think you should add examples about how to execute the command, like `zathura` or `zathura {{path/to/file}}`, etc

**Comment by gsobell** [comment]
Besides opening file, navigation and zoom, what other shortcuts/commands should be mentioned?

Haven't used Adobe Acrobat in years, what other functionalities would casual PDF reader users need/use/expect?

File: pages/linux/zathura.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+`r`
+
+- Invert Colors:
+
+`Ctrl + R`
+
+- Search for text:
+
+`/`
```
Comment (blueskyson) [inline pages/linux/zathura.md:25]: Does this support regex or just token?

```
/{{token}}
/{{regular_expression}}
```

Comment (gsobell) [reply pages/linux/zathura.md:25]: I believe only strings, no regex.

Comment (blueskyson) [inline pages/linux/zathura.md:25]:
```suggestion
- Search for text by given string:

`/{{string}}`
```

**Review by blueskyson — APPROVED** [review]
Thanks for your contribution.

**Review by owenvoke — APPROVED** [review]

File: pages/linux/zathura.md (hunk 6 above)
Comment (sbrl) [reply pages/linux/zathura.md]: I think it's probably best as a separate example.

**Review by sbrl — APPROVED** [review]
Looks good - thanks @gsobell!

**Pull request merged**